test/src/org/labkey/test/components/targetedms/UtilizationCalendarWebPart.java

```diff
@@ -22,7 +22,7 @@ public UtilizationCalendarWebPart(WebDriver driver)
 
     public void waitForPage()
     {
-        getWrapper().waitFor(() -> elementCache().yearTitle.isDisplayed(this),
+        WebDriverWrapper.waitFor(() -> elementCache().yearTitle.isDisplayed(this),
                 "Calendar did not load on time",
```

Comment:
What is the difference between getWrapper wait and WebDriverWrapper wait?

Reply:
`WebDriverWrapper.waitFor` is a static method. Referencing it through the non-static `getWrapper` is allowed in Java but is misleading.